lib/melodica_inventory_web/controllers/item_controller.ex

```diff
@@ -48,17 +48,30 @@ defmodule MelodicaInventoryWeb.ItemController do
   defp add_new_item(item_params) do
     Multi.new()
     |> Multi.insert(:item, Item.changeset(%Item{}, item_params))
-    |> Multi.run(:image, fn state -> upload_image(state, item_params["image"]) end)
+    |> Multi.run(:image, fn state -> upload_images(state, item_params["image"]) end)
   end
 
-  defp upload_image(%{item: %Item{}}, nil), do: {:error, "You need to upload an image"}
+  defp upload_images(%{item: %Item{}}, nil), do: {:error, "You need to upload an image"}
 
-  defp upload_image(%{item: %Item{id: item_id}}, %Plug.Upload{path: filename}) do
+  defp upload_images(%{item: %Item{id: item_id}}, uploaded_files) do
+    Enum.map(uploaded_files, &upload_to_cloudex_and_save(&1, item_id))
+    |> format_response
+  end
```

Comment:
This function seems to be duplicated across the controllers. Extracting it into a module in some context is going to fix that.